src/fiber/network.rs

```diff
@@ -1576,6 +1567,10 @@ where
                         self.network_graph.write().await.add_node(node_info);
 
                         // TODO: bookkeeping how many nodes we have connected to. Stop connnecting once we surpass a threshold.
+                        let peer_id = node_announcement.peer_id();
+                        state
+                            .persistent_state
+                            .save_peer_addresses(peer_id, node_announcement.addresses.clone());
```

Comment:
should we save the data in DB after insert new one?
from the current code, seems we only save all the data in rocksdb when `post_stop` called,
i'm thinking if there fiber node is killed forcely, we can not get the data back after restarting?
since `save_peer_addresses` only save it in memory.